luigi_pipeline/lib/model/sv_mt_schema.py

```diff
@@ -0,0 +1,225 @@
+import hail as hl
+
+from lib.model.base_mt_schema import row_annotation, RowAnnotationOmit
+from lib.model.seqr_mt_schema import BaseVariantsSchema, SeqrGenotypesSchema, SeqrVariantsAndGenotypesSchema
+
+from hail_scripts.computed_fields import variant_id
+
+
+BOTHSIDES_SUPPORT = "BOTHSIDES_SUPPORT"
+GENE_SYMBOL = "gene_symbol"
+GENE_ID = "gene_id"
+MAJOR_CONSEQUENCE = "major_consequence"
+PASS = "PASS"
+
+# Used to filter mt.info fields.
+CONSEQ_PREDICTED_PREFIX = 'PREDICTED_'
+NON_GENE_PREDICTIONS = {'PREDICTED_INTERGENIC', 'PREDICTED_NONCODING_BREAKPOINT', 'PREDICTED_NONCODING_SPAN'}
+
+
+INTERVAL_TYPE = 'array<struct{type: str, chrom: str, start: int32, end: int32}>'
+
+
+def get_cpx_interval(x):
+    # an example format of CPX_INTERVALS is "DUP_chr1:1499897-1499974"
+    type_chr = x.split('_chr')
+    chr_pos = type_chr[1].split(':')
+    pos = chr_pos[1].split('-')
+    return hl.struct(type=type_chr[0], chrom=chr_pos[0], start=hl.int32(pos[0]), end=hl.int32(pos[1]))
+
+
+class SeqrSVVariantSchema(BaseVariantsSchema):
+
+    def __init__(self, *args, gene_id_mapping=None, **kwargs):
+        super().__init__(*args, **kwargs)
+        self._gene_id_mapping = gene_id_mapping
+
+    def sv_types(self):
+        return self.mt.alleles[1].replace('[<>]', '').split(':', 2)
+
+    @row_annotation()
+    def sc(self):
+        return self.mt.info.AC[0]
```

Comment:
Sorry I forgot hail just imports AC as an array. I got confused because hl.agg.callstats produces an array with AC = [ref AC, alt AC] so wanted to make sure.